pandas/core/generic.py

```diff
@@ -2180,6 +2178,141 @@ def to_excel(
         freeze_panes: tuple[int, int] | None = None,
         storage_options: StorageOptions | None = None,
         engine_kwargs: dict[str, Any] | None = None,
+        autofilter: bool = False,
+    ) -> None:
+        """
+        Write object to an Excel sheet.
```

Comment:
Can you remove the doc decorator on L2149

Reply:
Also - it looks like there was a merge issue here and code got duplicated below.